packages/frontend/src/js/app/store/index.js

```diff
@@ -16,230 +15,247 @@ import activities from "./modules/activities";
 import locations from "./modules/locations";
 import directory from "./modules/directory";
 
-Vue.use(Vuex);
-
-export default new Vuex.Store({
-    modules: {
-        activities,
-        locations,
-        directory
-    },
-    state: {
-        towns: {
-            data: [],
-            loading: true,
-            error: null,
-            sort: "updatedAt",
-            filters: {
-                population: [],
-                fieldType: [],
-                justice: [],
-                origin: [],
-                conditions: [],
-                status: "open",
-                location: null,
-                actors: [],
-                target: [],
-                search: ""
-            },
-            currentPage: 1
-        },
-        detailedTown: null
-    },
-    mutations: {
-        setLoading(state, value) {
-            state.towns.loading = value;
-        },
-        setTowns(state, towns) {
-            state.towns.data = towns;
-        },
-        setError(state, err) {
-            state.towns.error = err;
-        },
-        setSort(state, sort) {
-            state.towns.sort = sort;
-        },
-        setFilters(state, filters) {
-            state.towns.filters = filters;
-        },
-        setLocation(state, location) {
-            state.towns.filters.location = location;
-        },
-        setCurrentPage(state, page) {
-            state.towns.currentPage = page;
+export default function(Vue) {
```

Comment:
Le fichier main.js expose une fonction avec Vue en paramètre. J'ai du transformer l'initialisation du store en fonction pour pouvoir définir Vue.use(Vuex) depuis le Vue passé en paramètre

Le seul impact est aux endroits dans le code ou on utilisait le store depuis ce fichier. Il faut maintenant passer par this.$store